miro_preprocessor/xml_to_json_converter/src/test_run.py

```diff
@@ -128,3 +128,12 @@ def _get_body(key):
     actual_json_objects = {key: _get_body(key) for key in keys}
 
     assert expected_json_objects == actual_json_objects
+
+
+def test_build_collection_id():
+    src_key = "source/Images-V.xml"
```

Comment:
I'm not sure whether that will cause issues in the transformer or not, but the files we've put in S3 so far have key `sources/images-V.xml`, without the capital `I`. So the expected collection, in that case, would be `images-V` instead of `Images-V`. The logic in the transformer checks for `Images-V` with capital `I` if I remember correctly, so _I think_ we would probably still have issues with this logic. Maybe the transformer should be a bit more robust in wjat it looks for

Reply:
👍  https://github.com/wellcometrust/platform/pull/974/files#diff-6e1939a684d42b780d5ea33c26b2aab8R22